Formula/qgis3.rb

```diff
@@ -525,7 +555,7 @@ def install
     # args << "-DWITH_BINDINGS=ON"
 
     # python Configuration
-    args << "-DPYTHON_EXECUTABLE='#{`python3 -c "import sys; print(sys.executable)"`.chomp}'"
+    args << "-DPYTHON_EXECUTABLE='#{`#{HOMEBREW_PREFIX}/opt/python/bin/python3 -c "import sys; print(sys.executable)"`.chomp}'"
```

Comment:
This seems like a great fix. Perhaps we could also scan $PATH and remove any other python references?

Reply:
how would you do the PATH scan and remove the reference?

Reply:
You could probably use a regex to look for any path that contains a `python` reference and simply remove those paths. Let's see if this improves things, and we can try that at a later date.